src/utils/Logger.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+/**
+ * Native interface
+ *
+ * @public
+ */
+export interface LoggerImplementation {
```

Comment:
i do not like the name `LoggerImplementation` its confusing as it is not an implementation, but an interface that a logger implementation must fullfill...